day-1/slides-draft.md

```diff
@@ -0,0 +1,332 @@
+# N8N Tutorial - Day 1 (DRAFT STRUCTURE)
+
+---
+
+## [TO REVIEW] SECTION 1: Introduction & Foundation Theory
+
+### Slides yang ada (KEEP):
+- Apa itu N8N? (slide 12-24)
+- Target Hari Ini (slide 27-36)
+
+### Slides yang DITAMBAHKAN:
+- **[TO REVIEW] Apa itu Automation?**
+  - Definisi automation untuk bisnis
+  - Contoh manual process vs automated process
+  - Business value: time saved, error reduction
+
```

Comment:
Maunya action dulu deh baru trigger, terus nanti perlu ada semacam tugas gitu, triggernya jadinya yang ini

Tugas 1:
- Buat workflow baru
- Buat triggernya "Execute
workflow"
- Action yang dilakukan:
    - Menambahkan baris baru
ke google sheet berupa 
angka yang diinginkan "975"
Jalankan “Execute workflow” kalau sudah done

Prereq:
- D google sheet -> Oauth2 (Sign In With Google)
  success = "Account connected"
- Bikin blank spreadsheet “Workshop N8N” di sheets.google.com, buat 
sheet baru dengan judul "Angka", lalu buat header column
berupa "Angka"

Target:  Ada angka yang diharapkan muncul di row baru di google sheet yang kita punya

Reply:
Jadi biar nangkep dulu bisa automate kita bertahap, dari sekalipun harus klik execute workflownya manual tapi sisanya automate. Untuk tugas ini taruh di docs baru juga kali yak

Reply:
Jadi kalau bisa mereka enggak denger term trigger kecuali enggak lama bener2 di deep dive action terhadap hal tersebut